source/js/theme.js

```diff
@@ -4,15 +4,23 @@ const getTheme = window.localStorage && window.localStorage.getItem("theme");
 const themeToggle = document.querySelector(".theme-toggle");
 const isDark = getTheme === "dark";
 
+function changeTheme(themed, dark) {
```

Comment:
Why do we need this?

Reply:
are you suggesting that it would be possible to do what i wanted without editing the theme source at all?
this is not my area of expertise so any feedback you have (including to just throw this all away!) is welcome 😅